angular/core/services/connection_service.coffee

```diff
@@ -0,0 +1,8 @@
+angular.module('loomioApp').factory 'ConnectionService', ->
+  new class ConnectionService
```

Comment:
I don't quite follow the need for this class yet. I think this might belong in MessageChannelService, under the 'subscribe' method

```
new class MessageChannelService
  subscribe: (options = {}) ->
    $http.post('/api/v1/message_channel/subscribe', options).then handleSubscriptions
    PrivatePub.faye (client) ->
      client.on 'transport:up', ->
        FlashService.warning 'You're disconnected!'
      client.on 'transport:down', ->
        FlashService.success 'We're back!'
        Records.events.fetchNew()
```

It might even go into the 'handleSubscriptions' method, ie once we know that we've successfully subscribed to our channels, then we apply our onConnect and onDisconnect methods